.github/workflows/sbom_generator.yaml

```diff
@@ -0,0 +1,73 @@
+# Generate SBOM for latest version of dotCMS and put into core-test-repo
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      dotcms_version:
+        description: 'The dotCMS version tag (e.g., v24.07.30)'
+        required: true
+        default: 'v24.07.30'
```

Comment:
Is this the starting point of version releases / default release that we want SBOM generated for it?

Reply:
I'd think we don't want to set the default top a version for this case because if a version isn't provided there's no value in the run.  It could create a state where there's no failures, but the sbom isn't generating what we think it is.

Reply:
Great. Does that also mean there should be no problem or adverse effect from removing default?

Reply:
Ok did some quick digging. Seems to be a common and years old challenge with gh actions. Not clear if it affects workflows too. But according to [docs](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions?t&utm_source=perplexity#example-specifying-inputs), required inputs that aren't provided still don't throw a good error. 


There's debate out there and I didn't go deep but some recommendations to default to an empty string or `no version provided` _(I think this is my preference)_ as the string so that troubleshooting is easier when it breaks in the future.  Here's the [ref issue](https://github.com/orgs/community/discussions/26777) for it.